visibility.go

```diff
@@ -0,0 +1,38 @@
+package chromedp
```

Comment:
this file is tiny, I would not add it. just add the funcs elsewhere like util.go, even though I hate that name.

Reply:
why are small files a problem?